CONTRIBUTING.md

```diff
@@ -40,49 +38,54 @@ You need to fork [scratchy repository](https://github.com/mdl29/scratchy)
 **see all labels [here](https://github.com/mdl29/scratchy/labels) and their meaning**
 
-4. Add your issue to a project the current one ([V1](https://github.com/mdl29/scratchy/projects/1)) if it is important or the next one if not
+5. Add your issue to a project the current one ([V1](https://github.com/mdl29/scratchy/projects/1)) if it is important or the next one if not
 because in the future we will go over [V2](https://github.com/mdl29/scratchy/projects/2)
  
-5. Add your issue to the current one milestone if it is important or the next one.
+6. Add your issue to the current one milestone if it is important or the next one.
+
+* **Example :** `[example] Create scratchy service that contains methods for backend requests access`
+    ![example issue](doc/example/example-issue.png)
   
 
 ## **Create a pull request :**  
 
 When you think your code is ready to be merged to the project create a pull request.
 
-1. choose the title:
-    when your pull request is ready link to an issue :
-
-    * **example :**` Resolves #ISSUEID : title of the issue `
+1. Choose the title:
+    patern: `ISSUEID title of the issue `
+    (if you have respected the naming branch pattern the title will be good automatically)
 
     If you want to make a pull request to add new features but you don't have an issue, you must be create an issue and in them explain what you want add,resolves...  Finally after you have resolved her you can push your pull request respecting issue and pull request template. 
  
-2. make a great description :
+2. Make a great description :
     
-    Please write a pull request description like this :
-      ![example](./doc/example/example-description.png)
-    The description must have
-    the id of the issue resolved and a message who say what you add or do in your pull request.
-    > ℹ️ [how to link a issue and use the keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)(Fixes,resolves...)
+    Please write a pull request description like with :
+        - the id of the issue resolved with a keyword to link to the coresponding issues
+          > ℹ️ [how to link a issue and use the keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)(Fixes,resolves...)
+        - a message who say what you add or do in your pull request.
 
-    Add [labels](https://github.com/mdl29/scratchy/labels) (Frontend,Backend,documentation...), add to project your pull request(V1,V2) and milestone depending your issue.
-    
-* **Real Example :**   
-   * if you want to solve #91 issue you should have named your pull request : `resolves #91 : Create a scratchy service` 
-  ![example-issue](./doc/example/example_pr_title.png)
+3. Assign it to yourself
```

Comment:
What if the reporter doesn't know how to solve it ^^

Reply:
oops

Reply:
wait it is a pull request so if you write it you are the one that solved the problem